### ISSUE TITLE
Pr curve uses interpolation (aka is wrong)

### ISSUE THREAD

**Issue by amueller** [INRIA/scikit-learn-mooc#80]
Here:
https://inria.github.io/scikit-learn-mooc/python_scripts/metrics.html#evaluation-and-different-probability-thresholds

Luckily ``plot_precision_recall_curve`` is released now ;)

**Comment by glemaitre** [comment]
You can see the comment:

```
# # FIXME: to be used when solved in scikit-learn
# from sklearn.metrics import plot_precision_recall_curve

# disp = plot_precision_recall_curve(
#     classifier, X_test, y_test, pos_label='donated',
# )
```

We sill have the issue of the positive label if I am right :). We need 0.24.
But this is on the TODO list in 0.24.

**Comment by glemaitre** [comment]
Oh sorry, I did not see the title of the issue actually. We should solve that temporarily then. Thanks @amueller

**Comment by amueller** [comment]
I saw ;) I think the pos_label change is not really required, if you want to use any built-in metrics you have to convert the labels to boolean anyway, which is the way I teach it. Otherwise you have two wrap everything with make_scorer and there's really no benefit over using ``y_train_bin = y_train == pos_label``.

**Comment by glemaitre** [comment]
fixed by updating to scikit-learn 0.24 and the right helper
